django_rq/views.py

```diff
@@ -223,6 +223,43 @@ def deferred_jobs(request, queue_index):
     return render(request, 'django_rq/jobs.html', context_data)
 
 
+@staff_member_required
+def scheduled_jobs(request, queue_index):
+    if not RQ_SCHEDULER_INSTALLED:
+        return redirect('rq_jobs', queue_index)
+
+    queue_index = int(queue_index)
+    queue = get_queue_by_index(queue_index)
+
+    scheduler = get_scheduler(queue.name)
+    all_jobs = [job for job in scheduler.get_jobs()
```

Comment:
This should just display all scheduled jobs (see [comment](https://github.com/rq/django-rq/pull/276/files#r173675281)).